src/opnsense/mvc/app/controllers/OPNsense/Unbound/Api/ServiceController.php

```diff
@@ -44,7 +44,11 @@ public function dnsblAction()
         $this->sessionClose();
         $backend = new Backend();
         $backend->configdRun('template reload ' . escapeshellarg(static::$internalServiceTemplate));
-        $response = $backend->configdRun(static::$internalServiceName . ' dnsbl');
-        return array('status' => $response);
+        $response = json_decode(trim($backend->configdRun(static::$internalServiceName . ' dnsbl')), true);
+        if ($response !== null) {
+            $response['status'] = "OK";
```

Comment:
what sort of output are we looking for here? seems like a lot to emulate "OK". Better to add a separate configd action for it?

Reply:
specifically, the amount of RRs added/deleted from the running Unbound instance. It would be nice to include the file stats generated in the blocklists as well, but this is all from a UX perspective and up for debate.

I agree the emulation is a bit out of place, either we scrap showing the user anything and make it a simple script or include the status key in the wrapper output since the SimpleActionButton always requires one.

Reply:
> scrap showing the user anything and make it a simple script

well you can leave the output but I would change "scritp_output" to "script" in the template to get rid of this workaround

Reply:
how would I fetch the amount of records then? It's returned by the script itself.

Reply:
ok fair enough, but shouldn't we at least try to make the return consistent? status + status_message vs. message at the moment (see below)

Reply:
We can move "status":"OK" to the script output, but "status_msg" needs a gettext() so it needs a modification anyway.